vllm/lora/worker_manager.py

```diff
@@ -174,9 +186,15 @@ def _load_lora(self, lora_request: LoRARequest) -> LoRAModel:
     def add_dummy_lora(self, lora_request: LoRARequest, rank: int) -> bool:
         if lora_request.lora_int_id in self.list_loras():
```

Comment:
Add assert self._cached_dummy_lora is not False here?

Reply:
that would make caching mandatory

Reply:
oh do we have other use case other than using it for profiling run (which already uses cache)?

Reply:
nope